core/src/main/java/org/infinispan/notifications/cachemanagerlistener/CacheManagerNotifierImpl.java

```diff
@@ -133,7 +135,10 @@ private void invokeListener(ListenerInvocation listener, EventImpl e) {
       } catch (Exception x) {
          // Only cache entry-related listeners should be able to throw an exception to veto the operation.
          // Just log the exception thrown by the invoker, it should contain all the relevant information.
-         log.error(x);
```

Comment:
JGroups has a neat trick, refusing to log repeated messages more than once a second (I think). Unfortunately, it also has this habit of only logging the exception message, and it makes it very hard to figure out what's going on some of the time.

BTW, since this is an error message, it should be i18n-ed in the `Log` interface.